automation/service/session.go

```diff
@@ -272,6 +274,29 @@ func (svc *session) Watch(ctx context.Context) {
 		//svc.suspendAll(ctx)
 	}()
 
+	// Prompt routine
+	go func() {
+		defer sentry.Recover()
+		defer promptTicker.Stop()
+		defer svc.log.Info("stopped")
+
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			case <-promptTicker.C:
+				activeSessions := websocket.GetActiveSessions()
+				for _, s := range activeSessions {
+					var ctxr = context.Background()
+					pp := svc.PendingPrompts(auth.SetIdentityToContext(ctxr, s.User()))
+					if len(pp) > 0 {
+						_ = s.Send(websocket.Message(websocket.StatusOK, websocket.WorkflowApplication, pp), s.User().Identity())
```

Comment:
the more I think about this application filter, the less sense it makes... 
either way, this info should come from the prompt itself and you should not hard coded it like this.